logfile

```diff
@@ -192,3 +192,53 @@ STATEMENT:  CREATE DATABASE "daweidai_development" ENCODING = 'unicode'
 ERROR:  database "daweidai_development" is being accessed by other users
 DETAIL:  There is 1 other session using the database.
 STATEMENT:  DROP DATABASE IF EXISTS "daweidai_development"
+LOG:  database system was shut down at 2015-10-13 09:11:44 PDT
```

Comment:
You have committed it to your git local repository. So something like `git rm logfile` - recommit and push should get rid of that.